cli/cmd/uninstall.go

```diff
@@ -36,9 +36,16 @@ var uninstallCmd = &cobra.Command{
 		}
 		ctx := cmd.Context()
 
+		errMessage := fmt.Errorf("Odigos is not installed in the cluster")
```

Comment:
Should we use `fmt.Errorf`? Can we use plain string instead?

Reply:
Hey @blumamir, I want to use something like l.Error(errMessage) but i am not able to do it
- Initially i want to proceed with ```l.Error(errMessage)``` but when i use that it says ```l not defined```
- So i left it there.
- I have changed the code you can see at the end of this thread.